org.zealdocs.Zeal.json

```diff
@@ -0,0 +1,41 @@
+{
+  "id": "org.zealdocs.Zeal",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.11",
+  "sdk": "org.kde.Sdk",
+  "base": "io.qt.qtwebkit.BaseApp",
+  "base-version": "5.11",
+  "command": "zeal",
+  "rename-icon": "zeal",
+  "finish-args": [
+    "--socket=wayland",
+    "--socket=x11",
+    "--share=ipc",
+    "--device=dri",
+    "--share=network"
+  ],
+  "cleanup": [
+    "/mkspecs",
+    "/include",
+    "/lib/cmake",
+    "/lib/mkspecs"
+  ],
+  "modules": [
+    {
+      "name": "zeal",
+      "buildsystem": "cmake-ninja",
+      "config-opts": [
+        "-DCMAKE_BUILD_TYPE=Release",
+        "-DQt5WebKit_DIR=/app/lib/cmake/Qt5WebKit",
+        "-DQt5WebKitWidgets_DIR=/app/lib/cmake/Qt5WebKitWidgets"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/zealdocs/zeal/archive/v0.6.1.tar.gz",
```

Comment:
Please use `https://github.com/zealdocs/zeal/releases/download/v0.6.1/zeal-0.6.1.tar.[gz|xz|zip]` instead of the automatically created tarballs (GitHub does not guarantee their immutability).

Reply:
Interesting, never encountered any issue in the wild. Thanks!

Reply:
I got a ticket about this a while back: zealdocs/zeal#897.